build_diffoscope.sh

```diff
@@ -7,15 +7,21 @@ diffoscope_file=$(dirname ${compare})/$(basename ${compare} .buildinfo.compare).
 
 echo -e "saving build diffoscope file to \033[1m${diffoscope_file}\033[0m"
 
-grep '# diffoscope ' ${compare} | sed -e 's/# diffoscope //' | ( while read -r line
+set sed="sed"
+if [ "$(uname -s)" ==  "Darwin" ]
+then
```

Comment:
we can reuse the existence check pattern to bring nice error message:
```
command -v gsed >/dev/null 2>&1 || { echo "require GNU sed: brew install gsed  Aborting."; exit 1; }
```

Reply:
makes sense to me. done 2034885